src/kibana/components/courier/fetch/request/segmented.js

```diff
@@ -119,6 +119,15 @@ define(function (require) {
      *********/
 
     /**
+     * Set the sort total number of segments to emit
+     *
+     * @param {number}
+     */
+    SegmentedReq.prototype.setMaxSegments = function (maxSegments) {
+      this._maxSegments = _.parseInt(maxSegments);
```

Comment:
I thought we had a rule in jshint about adding base 10 to parseInt. In any event, you should do it.